.travis.yml

```diff
@@ -8,9 +8,12 @@ before_script:
 
 go: tip
 script:
+  - test -z "$(gofmt -s -l $(find ./arm/* -type d -print) | tee /dev/stderr)"
   - test -z "$(gofmt -s -l -w management | tee /dev/stderr)"
   - test -z "$(gofmt -s -l -w storage | tee /dev/stderr)"
   - go build -v ./...
+  - test -z "$(go vet $(find ./arm/* -type d -print) | tee /dev/stderr)"
```

Comment:
@ahmetalpbalkan Not that I could find. I tried various incantations before settling on this. I felt it poor form to list each package individually since it would mean having to update the file any time we added a new package.